packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -248,11 +235,22 @@ const EDiscovery = WebexPlugin.extend({
       })
         .then((res) => {
```

Comment:
Should this code handle the failure scenario in a `catch`?

Reply:
I don't believe so, see full answer below